dev/Deployment/DeploymentManager.cpp

```diff
@@ -112,24 +112,73 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         {
             // Build package family name based on the framework naming scheme.
             std::wstring packageFamilyName{};
+            // The main and singleton packages contain a shortened version tag for the package family name.
+            std::wstring formattedVersionTag{};
+
+            auto ExtractFormattedVersionTag = [](const std::wstring& versionTag, bool addDashPrefix = false) -> std::wstring
+            {
+                wchar_t firstLetter = versionTag.size() > 1 ? versionTag[1] : L'\0';
+                std::wstring numberBeforeUnderscore;
+                size_t i = 2;
+                while (i < versionTag.size() && iswdigit(versionTag[i]))
```

Comment:
Assuming the number comes after -experimental, it is going to stop on e at the start, isn't it?